source/FreeRTOS_IP_Utils.c

```diff
@@ -550,7 +554,7 @@ static void prvChecksumProtocolSetChecksum( BaseType_t xOutgoingPacket,
         *( pxSet->pusChecksum ) = pxSet->usChecksum;
     }
 
-    #if ( ipconfigHAS_DEBUG_PRINTF != 0 )
+    #if ( ipconfigHAS_DEBUG_PRINTF == 0 )
```

Comment:
As suggested offline, can you update this with proper return, instead of == 0 {which was a workaround}

Reply:
sure, updated